### ISSUE TITLE
Sandbox - Brak możliwości tworzenia przesyłki do zamówienia

### ISSUE THREAD

**Issue by inTestowy** [allegro/allegro-api#7223]
Na koncie testowym (intest2.testowy@gmail.com)  nie jesteśmy w stanie dodać żadnej przesyłki do zamówień, przykładowo:

![image](https://github.com/allegro/allegro-api/assets/111511140/266bde4b-a38a-4c53-9b97-d4926cee4db1)

czy mogę prosić o pomoc w tym temacie.

**Comment by MartaNowaczyk** [comment]
Jeśli dotyczy to zamówień z początku czerwca (do 7.06.) to powodem tego błędu jest brak synchronizacji danych na Sandboxie. W takim wypadku trzeba utworzyć nowe zamówienia do testów.

**Comment by inTestowy** [comment]
Wczoraj stworzonych zamówień też to dotyczy ?

Bo do wczorajszego zamówienia dopiero w dniu dzisiejszym istnieje możliwość utworzenia przesyłki.

**Comment by MartaNowaczyk** [comment]
Wczorajszych nie powinno to dotyczyć. Jeśli błąd nadal się pojawia dla nowych zamówień, to możesz podać ID tego zamówienia do weryfikacji.

**Comment by inTestowy** [comment]
W tej chwili dla nowych jest ok.

Proszę mi jeszcze wyjaśnić, w takim razie mam rozumieć, że tylko do 7 dni wstecz synchronizowane są dane w Sandboxie ?

**Comment by MartaNowaczyk** [comment]
Nie, mieliśmy przeniesienie danych i tylko w tym wypadku wcześniejsze nie są po prostu zsynchronizowane.

**Comment by inTestowy** [comment]
ok, dziękuję za informację
